exchanges/orders.py

```diff
@@ -0,0 +1,146 @@
+from .assets import Asset
+
+def basic_order_check(action, asset_pair):
+    if action not in ['buy', 'sell']:
+        raise ValueError("Action must be 'buy' or 'sell'")
+
+    asset0, asset1 = asset_pair
+    if not isinstance(asset0, Asset) or not isinstance(asset1, Asset):
+        raise ValueError("Not recognized asset pair")
+
+
+class Order(object):
+    """This is the simplest type of order, given an action (i.e. BUY/SELL);
+       an assets_pair (A0, A1) and a volume, it represents:
+
+       BUY/SELL `volume` of A0 using/for some volume of A1, which follows the
+       relationship: volume(A0)*price = volume(A1). Where `price` is dictated
+       by the market
+
+       Examples
+       --------
+       Order(action:BUY, asset_pair:(ETH, BTC), volume:1.5) means:
```

Comment:
I prefer my examples to be valid code, something that can be copypasted to ipython and works. Just my opinion, no need to change it.